com.opcoach.genmodeladdon.core/xtend-gen/com/opcoach/genmodeladdon/core/GMAConstants.java

```diff
@@ -36,5 +36,5 @@
   
   static final String DEFAULT_SRC_CLASS_IMPL_PATTERN = "{0}DevImpl";
   
-  static final String NATURE_ID = "com.opcoach.genmodeladdon.core.GMANature";
+  static final String NATURE_ID = "com.opcoach.genmodeladdon.ui.GMANature";
```

Comment:
core, not ui

Reply:
I Agree, then we must change nature Id inside ui plugin.xml for contribution menus (add/remove nature)